src/patternfly/components/Switch/examples/Switch.md

```diff
@@ -41,6 +41,28 @@ cssPrefix: pf-c-switch
 {{/switch}}
 ```
 
+### Label and check
+```hbs
+{{#> switch switch--attribute='for="switch-with-icon-1"'}}
+  {{#> switch-input id="switch-with-icon-1" switch-input--attribute='name="switchExample3" checked'}}{{/switch-input}}
```

Comment:
Looks like this is using the same `id` as the examples without a label. The `id` is what ties the `<label for="[id]">` to the `<input id="[id]">`, so you'll notice that if you toggle the "without label" switches, they control the input in this new "label and check" example.

Also since the input has label text, it needs `aria-labelledby=[id of label text]`

And looking at the react component, it doesn't include the `name` attribute by default since it isn't necessary, and we don't have it documented, so I think we can remove that here (and in the other examples).

```suggestion
  {{#> switch-input id="switch-label-check-1" aria-labelledby="switch-label-check-1-on" switch-input--attribute='checked'}}{{/switch-input}}
```

Reply:
Thank you for clarifying @mcoker  - it makes sense for the `id`s here need to be distinct here since they control the inputs.